src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -13,4 +13,35 @@
     <Compile Include="System.Runtime.CompilerServices.Unsafe.il" />
     <Reference Include="System.Runtime" />
   </ItemGroup>
+
+  <Target Name="AddResourcesFileToIlasm" BeforeTargets="CoreCompile" Condition="'$(OS)'=='Windows_NT'">
+    <MakeDir Directories="$(IntermediateOutputPath)Disassembly" />
```

Comment:
rather than a directory i'd just give it a semi-unique name (to preserve path chars).  Something like $(MSBuildProjectName).ref.il.